### PR TITLE
Define echeladder bonuses using enums

### PR DESCRIPTION
**Pull request by LunyaticCat**

### CODE REVIEW

File: src/main/java/com/mraof/minestuck/player/Echeladder.java
Comment (kirderf1) [inline src/main/java/com/mraof/minestuck/player/Echeladder.java:67]: Can be private final

**Review by kirderf1 — COMMENTED** [review]
Looks good. Just some style-related things.

File: src/main/java/com/mraof/minestuck/player/EnumEcheladderBonuses.java
Comment (LunyaticCat) [reply src/main/java/com/mraof/minestuck/player/EnumEcheladderBonuses.java:3]: sure

**Review by kirderf1 — APPROVED** [review]
It's good to go, but if you'd like, you could write a javadoc for the new enum before I merge.

**Pull request merged**
